code/payments/Billing-Test-Environment.md

```diff
@@ -0,0 +1,112 @@
+# Billing Test Environment
+
+This is a guide on how to setup a dev/test environment for the billing system of satellites.
+
+We will setup storj-sim and connect it to the staging Stripe and CoinPayments accounts.
+
+## Prepare the local code for testing
+
+1. Clone the following repositories, if you haven't done yet:
+  - https://github.com/storj/storj
+  - https://github.com/storj/tardigrade-satellite-theme
+2. Replace the cryptocurrency for payments from STORJ to [LTCT](https://www.coinpayments.net/help-testnet):
+```sh
+sed -i 's/coinpayments\.CurrencySTORJ/coinpayments.CurrencyLTCT/g' $HOME/storj/satellite/payments/stripecoinpayments/tokens.go
+```
+3. Copy the Tardigrade satellite theme:
+```sh
+cp -r $HOME/tardigrade-satellite-theme/europe-west-1/* $HOME/storj/web/satellite/
+```
+4. Build the satellite GUI:
+```sh
+cd $HOME/storj/web/satellite
+npm install
+npm run build
+cd $HOME/storj
+```
+5. Build the storagenode GUI:
+```sh
+cd $HOME/storj/web/satellite
```

Comment:
should this be `cd $HOME/storj/web/storagenode`?

Reply:
True. I removed this step because it does not seem to be necessary.